### ISSUE TITLE
Lack of people and public interest

### ISSUE THREAD

**Issue by drahnr** [drahnr/oregano#161]
Since I am entangled in more than just one new endeavor, I am looking for someone to finish the gtk3 adjustments or for contributors and potential future maintainers.

If nobody steps up, this project has to be considered dead.

**Comment by junaidnaseer** [comment]
Hi,

Are you still looking for a maintainer?

**Comment by drahnr** [comment]
Yes, I am looking for someone who knows. Do you have experience with Gtk+3.x?

**Comment by drahnr** [comment]
Last call. Archiving on March 28th.

**Comment by drahnr** [comment]
@wereii && @surprized if you want to make this project live on, I am up to maintaining the CI + packaging, as well as mentoring. But somebody has to do the actual "product" development :upside_down_face:

**Comment by surprized** [comment]
Really wish I could help, but I don't think I have sufficient coding knowledge or the time to help right now. :(

**Comment by wereii** [comment]
I am afraid but I have minimal exposure to the usage of this software - I am way more of an software guy than electronics. Would love to keep this afloat but I don't think I am the right guy.
